.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.3.1
```

Comment:
You don't really need to be bumping this version since we already have a bumped version in `develop` and that is only really be released once we merge to master. Until then, these versions are not really released, so after the first bump get's into `develop`, all the later ones are only kind of being scheduled to be released together.

Reply:
Oh sorry about that, that makes sense